week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,25 +19,39 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(formatig) {
+  const tax = formatig * 0.2 + formatig;
+  return `£${tax.toFixed(2)}`;
+}
```

Comment:
You didn't read carefully the instruction. You don't need to do again the calculation, the function above done for you. It needs to be fixed.